neutrino.go

```diff
@@ -66,6 +67,10 @@ var (
 	// DisableDNSSeed disables getting initial addresses for Bitcoin nodes
 	// from DNS.
 	DisableDNSSeed = false
+
+	// DefaultFilterCacheSize is the number of filters neutrino will keep
+	// in memory if no size is specified in the neutrino.Config.
+	DefaultFilterCacheSize = 20
```

Comment:
Seems low? I would suggest 200, but again that's just taking a number out of the air